config/media.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Media Config
+ */
+
+return array(
```

Comment:
Pretty sure this could be combined with CDN config, but that all needs some rationalization now we could use Laravel Filesystem

Reply:
We should generally consider how we should logically group configs